README.md

```diff
@@ -1 +1,87 @@
-# functional-harness
+## testcafe-functional-harness
+
+Runs functional tests in local browsers and on SauceLabs.
+
+#### What it does
+
+1) Starts the server with the test pages.
+2) Opens the specified browsers (locally or on SauceLabs).
+3) Runs mocha tests.
+4) Closes the browsers and stops the server after all tests are finished.
+
+#### Install:
+
+```
+$ npm install testcafe-functional-harness
+```
+#### How to use:
+
+##### run-tests.js
+```js
+var createTestCafe            = require('testcafe');
+var TestCafeFunctionalHarness = require('testcafe-functional-harness');
+
+exports.browsersInfo = null;
+exports.testCafe     = null;
+
+var testCafeFunctionalHarness = new TestCafeFunctionalHarness();
+
+var functionalTestsBrowsers = [{
+    platform:    'Windows 10',
+    browserName: 'internet explorer',
+    alias:       'ie'
+}];
+
+var testSiteSettings = {
+    sitePort1: 2000,
+    sitePort2: 2001,
+    sitePath:  '/tests'
```

Comment:
`testSiteSettings.sitePort1`
It's not necessary to duplicate the `site` word. In this case we can use testSiteSettings.port1, ...
But I prefer to use constants here:
```js
var BROWSERS = [...]
var SITE_PORT1 = 2000;
var SITE_PORT2 = 2001;
var VIEWS_PATH = '/tests';
```